### PR TITLE
Web application interface

### PR DESCRIPTION
**Pull request by joanna-bukowska**

### CODE REVIEW

**Review by marcin-sieminski — COMMENTED** [review]
Maybe we should implement page layout also for small / mobile screens?

**Review by marcin-sieminski — APPROVED** [review]

File: DrinkManagerWeb/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -8,41 +8,54 @@
...
 
-    <footer class="border-top footer text-muted">
-        <div class="container">
-            &copy; 2020 - DrinkManagerWeb - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+            <div class="col-10 main-body">
+
+                <!-- page body -->
+                <div>
+                    @RenderBody()
+                </div>
```
Comment (wojciechkapko) [inline DrinkManagerWeb/Views/Shared/_Layout.cshtml]: This div element does not do anything, we could remove it.

**Review by wojciechkapko — COMMENTED** [review]
Most of the custom classes and their properties could be replaced with bootstrap classes for example:

for padding we could use boostrap "p" classes like p-3, pl-3 etc
for margin "m" classes
for text alignment text-left, text-center classes

File: DrinkManagerWeb/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (marcin-sieminski) [reply DrinkManagerWeb/Views/Shared/_Layout.cshtml]: I would consider leaving @RenderBody() with its div. RenderBody is a placeholder where all the page-specific views show up, wrapped in the layout page.

**Review by wojciechkapko — APPROVED** [review]

**Pull request merged**